ui.cpp

```diff
@@ -65,8 +87,12 @@ void Ui::update(unsigned long elapsed)
         }
     }
 
-    m_display.draw_bitmap(0, 0, Bitmap { 36, 64, DIGITS_36_64[m_big_number / 10] });
-    m_display.draw_bitmap(36, 0, Bitmap { 36, 64, DIGITS_36_64[m_big_number % 10] });
+    if (m_big_number != 0) {
+        if (m_big_number / 10 != 0) {
+            m_display.draw_bitmap(0, 0, Bitmap { 36, 64, DIGITS_36_64[m_big_number / 10] });
+        }
+        m_display.draw_bitmap(36, 0, Bitmap { 36, 64, DIGITS_36_64[m_big_number % 10] });
+    }
```

Comment:
Why are the conditionals necessary?

Reply:
Removed (entire pico ui) in 03d51eeef524e32c61277c2647ff848bb0dc321d

Reply:
This is probably UI-related: We need some sort of visual indicator, alerting us if the sensor values are old due to an unconnected sensor. Hence the outer condition. The inner one, I will remove.

Reply:
Added visual indication with double dash `--` in the latest version.